shared/src/config.rs

```diff
@@ -34,7 +34,7 @@ pub(crate) static CONFIG_SHARED: config_shared::Config = {
     load()
 };
 
-pub(crate) static CONFIG_SHARED_CRYPTO: OnceCell<ConfigCrypto> = OnceCell::new();
+pub static CONFIG_SHARED_CRYPTO: OnceCell<ConfigCrypto> = OnceCell::new();
```

Comment:
```suggestion
pub(crate) static CONFIG_SHARED_CRYPTO: OnceCell<ConfigCrypto> = OnceCell::new();
```

Provide getter method in shared instead